src/scancode/cli.py

```diff
@@ -785,4 +776,4 @@ def save_results(scanners, only_findings, files_count, results, format, options,
                     raise e
         return
 
-    write_formatted_output(scanners, files_count, version, notice, results, format, options, input, output_file, echo_stderr)
+    write_formatted_output(files_count, version, notice, results, format, options, input, output_file, echo_stderr, scan_output_plugins)
```

Comment:
I do not like that you pass a global here to an inner function call.... what exactly is held in `scan_output_plugins` ?